container/cypress/e2e/test-app/wc/wc-container.cy.js

```diff
@@ -131,5 +131,15 @@ describe('Web Container Test', () => {
             });
         });
     });
+    it('sendCustomMessage', () => {
+      cy.get('#sendCustomMessageBtn')
+        .click()
```

Comment:
try making a chained **.then** call just like in the updateContext test above. 🤔